pkg/apis/complianceoperator/v1alpha1/compliancescan_types.go

```diff
@@ -4,14 +4,15 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
```

Comment:
Is there DeepCopy stuff to regenerate with these changes?

Reply:
huh?

Reply:
I guess it's not needed, because it's just strings underneath. At least explicitly running `operator-sdk generate k8s` doesn't produce any changes.